openapi.go

```diff
@@ -425,12 +425,19 @@ func parseStructTags(t reflect.Type, schemaRef *openapi3.SchemaRef) {
 		example, ok := field.Tag.Lookup("example")
 		if ok {
 			propertyValue.Example = example
-			if propertyValue.Type.Is(openapi3.TypeInteger) {
+			switch {
+			case propertyValue.Type.Is(openapi3.TypeInteger):
 				exNum, err := strconv.Atoi(example)
 				if err != nil {
 					slog.Warn("Example might be incorrect (should be integer)", "error", err)
 				}
```

Comment:
Did you notice I face so much fun with integer ?

https://dev.to/ccoveille/about-the-gosec-g115-drama-or-how-i-faced-back-integer-conversion-overflow-in-go-1302

Is it why you ask me to check?

Here I would change the error message based on the error you receive from Atoi

https://pkg.go.dev/strconv#pkg-variables

For me you should look specifically for  ErrRange to report an issue about number is too big.

You may also simply catch the NumError and deal with it

https://pkg.go.dev/strconv#NumError

Same for float64 (the process is the exact same in all strconv)

Reply:
You can also use my lib to do that for you and get a clear error message. But I'm just pushy, because I like the lib I created. You don't have to use it here. You are dealing with 2 types only.

Reply:
I did not. But that definitely increases the scope of this PR. I'll make the changes this weekend.

Reply:
Reminder to myself to add some slogasserts.